src/Funding.sol

```diff
@@ -222,9 +215,15 @@ contract Funding is GlobalAccessControlManaged, ReentrancyGuardUpgradeable {
      * @param _assetAmountIn Amount of `asset` to exchange
      * @return xCitadelAmount_ Amount of `xCitadel` received at current price per share
      */
-    function getStakedCitadelAmountOut(uint256 _assetAmountIn) public view returns (uint256 xCitadelAmount_) {
```

Comment:
I notice the addition of paranthesis here. Is that added by the linter for clarity @sajanrajdev @shuklaayush

Reply:
Yep, these are all only formatting changes
```
(citadelAmount * ONE_ETH) /
            xCitadel.getPricePerFullShare();
```